packaging/Docker/Dockerfile.dev

```diff
@@ -1,11 +1,16 @@
-FROM fedora:34 AS base
+FROM fedora:36 AS base
+
+ARG CIM_VERSION=CGMES_2.4.15_16FEB2016
+ARG VILLAS_VERSION=c976cd62d8c6667a078be0785ca3e623a05f7456
```

Comment:
You updated VILLASnode to a later commit. Any specific reason for that? Can you mention this also in PR description (along with upgrade of Dockerfile to Fedora 36)?

Reply:
The new VILLASnode version contains fixes which I needed for building it on OPAL-RT targets.

I added a comment to the PR description.